frontend/packages/ux-editor/src/components/TaskNavigation/TaskCardEditing.test.tsx

```
@@ -0,0 +1,195 @@
+  beforeEach(() => {
+    confirmSpy = jest.spyOn(window, 'confirm');
+    confirmSpy.mockImplementation(jest.fn(() => true));
+  });
+
+  afterEach(() => {
+    confirmSpy.mockRestore();
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)